packages/database/src/schema.prisma

```diff
@@ -347,78 +299,163 @@ model AppealCase {
+  //***************************************************************************
+  // rule 6 fields
+  //***************************************************************************
 
-  appellantLinkedCase Boolean?
+  rule6StatementPublished      Boolean @default(false) /// todo: use date instead
+  rule6ProofsEvidencePublished Boolean @default(false) /// todo: use date instead
 
-  // questions relating to appeal documents (aapd-1126)
+  //***************************************************************************
+  // interested party fields
+  //***************************************************************************
 
-  updateDevelopmentDescription  String?
-  developmentDescriptionDetails String?
-  statusPlanningObligation      String?
+  interestedPartyCommentsPublished Boolean @default(false) /// todo: use date instead
 
-  // todo: there are many more fields! waiting on the model definition
+  //***************************************************************************
+  // Relations
+  //***************************************************************************
 
   /// link to root Appeal
   appealId String @unique @db.UniqueIdentifier
   Appeal   Appeal @relation(fields: [appealId], references: [id])
 
-  // Docs reference
-  Documents Document[]
+  LPAQuestionnaireSubmission LPAQuestionnaireSubmission?
 
-  // Neighbour address reference
-  NeighbouringAddresses NeighbouringAddress[]
+  Rule6Parties                    Rule6Party[]
+  AffectedListedBuildings         AppealCaseListedBuilding[]
+  Documents                       Document[]
+  NeighbouringAddresses           NeighbouringAddress[]
+  Events                          Event[]
+  AppealCaseLpaNotificationMethod AppealCaseLpaNotificationMethod[]
+}
 
-  // Linked cases
-  SubmissionLinkedCase SubmissionLinkedCase[]
+/// AppealCase relations
+/// maps all relations from a given caseReference to caseReference2
+/// linked|nearby
+/// linked: hierarchical + one directional, children (caseReference) will be linked to the lead (caseReference2)
+/// nearby: bi-directional, (A -> B) + (B -> A)
+/// No foreign keys as cases may not exist in FO yet
+model AppealCaseRelationship {
+  id   Int    @id @default(autoincrement())
+  type String @default("nearby") /// nearby|linked
+
+  caseReference  String
+  caseReference2 String
+}
 
-  // todo: indexes
+/// Lookup table for types of appeals processes, and appeal types such as D (Householder) and W (S78).
+/// { id: 1001, key: 'D', type: 'Householder', processCode: 'HAS' },
+model CaseType {
+  id          Int          @id
+  key         String       @unique
+  type        String       @unique
+  processCode String?      @unique
+  AppealCase  AppealCase[]
+}
 
-  // Questionnaires linked to this appeal case
-  LPAQuestionnaireSubmission LPAQuestionnaireSubmission?
+/// Lookup table for possible case procedure types.
+/// { key: 'written', name: 'Written' }
+model ProcedureType {
+  key         String       @id @unique
```

Comment:
should we use a separate ID for these lookup tables?
Currently id will match value sent in data model